perceval/backends/core/gitlab.py

```diff
@@ -0,0 +1,689 @@
+
+        reactions = []
+        self._push_cache_queue('{ISSUE-REACTIONS}')
+        self._flush_cache_queue()
+
+        if total_count == 0:
+            self._push_cache_queue('[]')
+            self._flush_cache_queue()
+            return reactions
+
+        group_reactions = self.client.issue_reactions(issue_number)
+
+        for raw_reactions in group_reactions:
+            self._push_cache_queue(raw_reactions)
+            self._flush_cache_queue()
+
+            for reaction in json.loads(raw_reactions):
+                reaction['user_data'] = self.__get_user(reaction['user']['login'])
+                reactions.append(reaction)
+
+        return reactions
+
+    def __get_issue_comments(self, issue_number):
+        """Get issue comments"""
+
+        comments = []
+        group_comments = self.client.issue_comments(issue_number)
+        self._push_cache_queue('{COMMENTS}')
+        self._flush_cache_queue()
+
+        for raw_comments in group_comments:
+            self._push_cache_queue(raw_comments)
+            self._flush_cache_queue()
+
+            for comment in json.loads(raw_comments):
+                comment_id = comment.get('id')
+                comment['user_data'] = self.__get_user(comment['user']['login'])
+                comment['reactions_data'] = \
+                    self.__get_issue_comment_reactions(comment_id, comment['reactions']['total_count'])
+                comments.append(comment)
+
+        return comments
+
+    def __get_issue_comment_reactions(self, comment_id, total_count):
+        """Get reactions on issue comments"""
+
+        reactions = []
+        self._push_cache_queue('{COMMENT-REACTIONS}')
+        self._flush_cache_queue()
+
+        if total_count == 0:
+            self._push_cache_queue('[]')
+            self._flush_cache_queue()
+            return reactions
+
+        group_reactions = self.client.issue_comment_reactions(comment_id)
+
+        for raw_reactions in group_reactions:
+            self._push_cache_queue(raw_reactions)
+            self._flush_cache_queue()
+
+            for reaction in json.loads(raw_reactions):
+                reaction['user_data'] = self.__get_user(reaction['user']['login'])
+                reactions.append(reaction)
+
+        return reactions
+
+    def __get_issue_assignee(self, raw_assignee):
+        """Get issue assignee"""
+
+        self._push_cache_queue('{ASSIGNEE}')
+        self._push_cache_queue(raw_assignee)
+        self._flush_cache_queue()
+        if 'username' in raw_assignee.keys():
+             assignee = self.__get_user(raw_assignee['username'])
+
+        return assignee
+
+    def __get_issue_assignees(self, raw_assignees):
+        """Get issue assignees"""
+
+        self._push_cache_queue('{ASSIGNEES}')
+        self._push_cache_queue(raw_assignees)
+        self._flush_cache_queue()
+        assignees = []        
+        for ra in raw_assignees:
+            if 'login' in ra.keys():
```

Comment:
I don't see in the GitLab API the attribute 'login' for assignees. If this is the case, the first if condition can be removed